www/webapp/vite.config.js

```diff
@@ -19,6 +19,14 @@ export default defineConfig({
     ],
     server: {
         port: 8080,
+        proxy: {
+            '/api': {
+                target: "https://desec.io",
+                changeOrigin: true,
+                secure: true,
+                // rewrite: path => path.replace('/api', ''),
+            }
+        }
```

Comment:
I usually run against the local dev API. I assume the above means that this would no longer work. Can we add a switch for this, e.g. via an env variable?

Reply:
Good point. I think this should be done with a separate PR.